meson_options.txt

```diff
@@ -0,0 +1,46 @@
+option('deprecated', type : 'boolean', value : false, description: 'Build deprecated components')
```

Comment:
I think this needs to default to `true` --- we still have some large projects which use the old vips7 API.